src/views/pages/items.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import ListenerComponent from '../../lib/listener-component.jsx';
+import ItemStore from '../../store/item-store.js';
+import { viewModel } from '../../lib/api-client.js'
+import { getModelsForNewItem } from '../../lib/api-client';
+import Item from '../components/item.jsx';
+export default class Models extends ListenerComponent {
+
+    getStores() {
+        return [
+            ItemStore
+        ];
+    }
+    getState() {
```

Comment:
I miss Miss Whitespace. :dagger::skull_and_crossbones: 

(and for the next two functions)